jax/core.py

```diff
@@ -1315,19 +1326,43 @@ def pp_eqn(eqn: JaxprEqn) -> PrettyPrint:
   else:
     return pp_lhs + pp_rhs.indent(2)
 
-def pp_jaxpr(jaxpr: Jaxpr, source_info: bool = False) -> PrettyPrint:
-  pp_outvars = str(tuple(jaxpr.outvars))
-  pp_eqns = map(pp_eqn, jaxpr.eqns)
+def pp_eqns(eqns: Sequence[JaxprEqn],
+            source_info: bool = False) -> Sequence[PrettyPrint]:
+  pp_eqns_ = map(pp_eqn, eqns)
```

Comment:
Maybe replace `pp_eqns_` with `pps` everywhere? I'm sure we can come up with a better and less typo-prone name :)